modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -1982,6 +1982,11 @@
        "HOSTNAME": {
          "LABEL": "Host name"
        },
+       "IS_PUBLISHED": {
```

Comment:
I don't think the old admin ui will automatically use this new filter anyway, so these translations string will go unused. Furthermore, the old admin ui is deprecated and will likely be removed with the next version of Opencast (see #5965). If you would like to make this filter available in the admin UI, consider the new admin ui instead https://github.com/opencast/opencast-admin-interface.

Reply:
I deleted the translation stings from the old admin ui, do I have to insert them in the new one somehow?

Reply:
If you know that the new admin ui will makes use of the translations strings, add them. If not, don't.